### PR TITLE
Translate 2.4.0-preview1 release post (ko)

### PR DESCRIPTION
**Pull request by riseshia**
Review please. ;_;

### CODE REVIEW

File: ko/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+title: "루비 2.4.0-preview1 릴리스"
+author: "naruse"
+translator: "shia"
+date: 2016-06-20 18:00:00 +0900
+lang: ko
+---
+
+루비 2.4.0-preview1의 릴리스를 알리게 되어 기쁘게 생각합니다.
+
+루비 2.4.0-preview1는 루비 2.4.0의 첫 번째 프리뷰입니다.
```
Comment (yous) [inline ko/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: 2.4.0-preview1는 → 2.4.0-preview1은

File: ko/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+그리고 여러분의 [느낀 점을 알려주세요](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)!
+
+## 2.3 이후의 주목할 만한 변경
+
+자세한 내용은 [뉴스](https://github.com/ruby/ruby/blob/v2_4_0_preview1/NEWS)와
+[변경기록](https://github.com/ruby/ruby/blob/v2_4_0_preview1/ChangeLog)을
+참고하세요.
+
+이러한 변경사항에 따라, 루비 2.3.0 이후로
+[파일 1140 개 수정, 33126 줄 추가(+), 50993 줄 삭제(-)](https://github.com/ruby/ruby/compare/v2_3_0...v2_4_0_preview1)가 이루어졌습니다.
```
Comment (yous) [inline ko/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: 1140 개 → 1140개
33126 줄 → 33126줄
50993 줄 → 50993줄

**Comment by yous** [comment]
I'm done.

**Comment by riseshia** [comment]
@yous Thanks, I just update!

File: ko/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+author: "naruse"
+translator: "shia"
+date: 2016-06-20 18:00:00 +0900
+lang: ko
+---
+
+루비 2.4.0-preview1의 릴리스를 알리게 되어 기쁘게 생각합니다.
+
+루비 2.4.0-preview1은 루비 2.4.0의 첫 번째 프리뷰입니다.
+이 첫 번째 프리뷰는 많은 새 기능과 개선들을 포함하고 있어서
```
Comment (marocchino) [inline ko/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: 첫 번째 could be omit in Korean.

File: ko/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+평소보다 이르게 릴리스 되었습니다.
```
Comment (marocchino) [inline ko/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: 이례적으로 better explain then 평소보다

File: ko/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+아직 기능들이 변경될 수 있으므로, 자유롭게
```
Comment (marocchino) [inline ko/news/_posts/2016-06-20-ruby-2-4-0-preview1-released.md]: How about `아직 기능이 확정되지 않았으니,` ?

**Comment by marocchino** [comment]
I'm done. 👯

**Comment by riseshia** [comment]
@marocchino Thanks! XD I applied your suggestion!

**Comment by marocchino** [comment]
Thanks! ❤️

**Pull request merged**